developer_manual/bugtracker/index.rst

```diff
@@ -14,6 +14,8 @@ Thank you for helping ownCloud by reporting bugs. Before submitting an issue, pl
 
 * If the issue is with the ownCloud server, report it to the `Core repository`_
 * If the issue is with the ownCloud client, report it to the `Client repository`_
```

Comment:
change 'client' to 'desktop client' then it's a 👍  :)

Happy new year.

Reply:
Sure. Thanks. Same to you.

Reply:
I've merged this ticket, as the requested change is different to the ticket's description. I'm making the change in a separate PR.